java-frontend/src/main/java/org/sonar/java/model/DefaultJavaFileScannerContext.java

```diff
@@ -104,7 +104,7 @@ public boolean fileParsed() {
   }
 
   /**
-   * @deprecated since SonarJava 5.12 - Use key of InputFile instead, using {@link #getInputFile()}.
+   * @deprecated since version 5.12 - Use key of InputFile instead, using {@link #getInputFile()}.
```

Comment:
Let's use `since plugin version` to be consistent with the others.

```suggestion
   * @deprecated since plugin version 5.12 - Use key of InputFile instead, using {@link #getInputFile()}.
```